tests/test_faker.py

```diff
@@ -94,3 +95,18 @@ def test_non_overlapping_dates(self, write_row_mock):
         """
         generate(StringIO(yaml), {}, None)
         assert row_values(write_row_mock, 0, "date") is None
+
+    @mock.patch(write_row_path)
+    def test_months_past(self, write_row_mock):
+        yaml = """
+        - object: A
+          fields:
+            date:
+              date_between:
+                start_date: -4M
+                end_date: -3M
+        """
+        generate(StringIO(yaml), {}, None)
+        the_date = row_values(write_row_mock, 0, "date")
+        assert (date.today() - the_date).days > 80
+        assert (date.today() - the_date).days < 130
```

Comment:
I like the ~10 day "I don't wanna think about variation in month lengths" fudge factor. :)